src/main/java/org/apache/cassandra/sidecar/metrics/instance/UploadSSTableMetrics.java

```diff
@@ -71,15 +79,21 @@ public UploadSSTableComponentMetrics(MetricRegistry metricRegistry, String sstab
             NamedMetric.Tag componentTag = NamedMetric.Tag.of("component", sstableComponent);
 
             rateLimitedCalls
-            = NamedMetric.builder(metricRegistry::meter)
+            = NamedMetric.builder(name -> metricRegistry.gauge(name, () -> new DefaultSettableGauge<>(0)))
```

Comment:
I'm not sure if rate limited calls are tied to a specific component. Is there a reason it matters to make it component specific?

Reply:
Moved them out. But I feel for highDiskUsage it might be valuable to keep component detail